addons/survey/wizard/survey_invite.py

```diff
@@ -75,7 +75,8 @@ def _get_default_author(self):
     @api.depends('partner_ids', 'survey_id')
     def _compute_existing_partner_ids(self):
         existing_answers = self.survey_id.user_input_ids
-        self.existing_partner_ids = existing_answers.mapped('partner_id') & self.partner_ids
+        partner_ids = set(self.partner_ids.ids)
+        self.existing_partner_ids = existing_answers.mapped('partner_id').filtered(lambda p: p.id in partner_ids)
```

Comment:
```suggestion
        self.partner_ids.browse(set(self.survery_id.user_input_ids.ids) & set(self.partner_ids.ids))
```
Actually maybe we can do something like this after speaking with flch :)

Reply:
Either way we can remove the 'mapped'

Reply:
> Actually maybe we can do something like this after speaking with flch :)

Thanks for the suggestion; indeed, it should be faster.
Note that I’ve replace “self.survery_id.user_input_ids.ids” by “self.survey_id.user_input_ids.partner_id.ids” to get partner ids and not answer ids.

Reply:
I'm not even convinced it's faster (because both creating and comparing the sets is linear time, and so is comparing each value to a set, so 3n vs 2n). But it's more succinct at least :)